### ISSUE TITLE
Custom fields

### ISSUE THREAD

**Issue by edmapd** [ZenHubIO/support#867]
Would be great for zenhub to offer the ability to add additional, custom fields to github issues. Ideally you could specify field type (text, pulldown multi-select, calendar, etc.), give it a field title, and then it would be added to your issue screen.

**Comment by azenMatt** [comment]
Hi @edmapd - thanks for the feedback - agreed this would be useful. 

Adding it into the backlog for everyone to 👍 and help prioritize

**Comment by HarveyEV** [comment]
Is there an ETA on this?

**Comment by kendramoroz** [comment]
@HarveyEV No exact ETA yet, but as more updates become available around this topic we'll be sure to post them!

**Comment by irahardianto** [comment]
Hey, how is it going with the custom field?

**Comment by kleighwe** [comment]
Pinging here! Any updates on custom fields?

**Comment by jmarks1992** [comment]
This is an enhancement that would make a world of difference for my team. Adding this (particularly with an API to modify the choices) and a creator filter would enable me to use ZenHub across our entire team, not just for engineers. In particular, it would make it possible for us to use it as part of a support function.

**Comment by jmarks1992** [comment]
This change (particularly if it were possible to change the custom fields via the Zenhub API) would make a huge difference to our workflow and would enable me to expand Zenhub to our support and customer success teams as well as our engineers.

**Comment by tdayton** [comment]
The absence of this capability is a major impediment to me having projects use ZenHub instead of JIRA.

**Comment by hliyan** [comment]
Agreed. I would even take something as simple and text-driven as this, as long as the field name is verified against a whitelist I define, and I can do queries based on them:

```
{
  "Component": "Mobile App",
  "Affected Version": "1.1.1",
  "Fix Version": "1.1.2",
  "Team": "XYZ"
}
```

**Comment by kmoore134** [comment]
+1 this would be a huge reason for us to use zenhub instead of our current solution.

**Comment by mikhail-denisenko** [comment]
Sitting in a room with product and project managers: trying to defend ZenHub but I'm running out of arguments: we need some simple extension on issue template for regulatory reasons and there's no traction from ZenHub on this matter. It is 99% likelihood ZenHub will unfortunately be decommissioned from the org in favor of "other tool" (guess which one)
